tests/turbomole/default.nix

```diff
@@ -0,0 +1,33 @@
+{ batsTest, turbomole } :
+
+batsTest rec {
+  name = "turbomole";
+
+  auxFiles = [
+    ./auxbasis
+    ./basis
+    ./control
+    ./coord
+    ./mos
+  ];
+  outFile = [
+    "ridft.out"
+    "ricc2.out"
+  ];
+
+  nativeBuildInputs = [ turbomole ];
+
+  # Turbomole writes to its input files, yes.
+  testScript = ''
+    @test "RI-ADC2" {
+      chmod +w auxbasis
+      chmod +w basis
+      chmod +w control
+      chmod +w coord
+      chmod +w mos
+      ${turbomole}/bin/ridft -smpcpus 2 > ridft.out
+      ${turbomole}/bin/ricc2 -smpcpus 2 > ricc2.out
+      grep "HOMO-LUMO gap:    0.48876" ridft.out
```

Comment:
Maybe a simple check also for ricc2?

Reply:
added the energy of one excited state as a test condition